content/en/docs/tutorials/cluster-management/kubelet-standalone.md

```diff
@@ -0,0 +1,521 @@
+{{< tab name="x86-64" codelang="bash" >}}
+curl -LO https://github.com/containernetworking/plugins/releases/download/v1.5.1/cni-plugins-linux-amd64-v1.5.1.tgz
+{{< /tab >}}
+{{< tab name="ARM64" codelang="bash" >}}
+curl -LO https://github.com/containernetworking/plugins/releases/download/v1.5.1/cni-plugins-linux-arm64-v1.5.1.tgz
+{{< /tab >}}
+{{< /tabs >}}
+
+Install:
+
+```shell
+sudo mkdir -p /opt/cni/bin
+```
+
+{{< tabs name="install_cni_plugins" >}}
+{{< tab name="x86-64" codelang="bash" >}}
+sudo tar Cxvf /opt/cni/bin/ cni-plugins-linux-amd64-v1.5.1.tgz
+{{< /tab >}}
+{{< tab name="ARM64" codelang="bash" >}}
+sudo tar Cxvf /opt/cni/bin/ cni-plugins-linux-arm64-v1.5.1.tgz
+{{< /tab >}}
+{{< /tabs >}}
+
+Quick test:
+
+```shell
+/opt/cni/bin/host-local --version
+```
+
+The output is similar to:
+
+```
+CNI host-local plugin v1.5.1
+CNI protocol versions supported: 0.1.0, 0.2.0, 0.3.0, 0.3.1, 0.4.0, 1.0.0
+```
+
+Configure:
+
+```shell
+sudo mkdir -p /etc/cni
+
+sudo tee /etc/cni/cni.template <<EOF
+{
+  "name": "containerd",
+  "cniVersion": "1.0.0",
+  "plugins": [{
+    "type": "ptp",
+    "ipMasq": true,
+    "ipam": {
+      "type": "host-local",
+      "subnet": "{{.PodCIDR}}",
+      "routes": [
+        {"dst": "0.0.0.0/0"}
+      ]
+    }
+  },{
+    "type": "portmap",
+    "capabilities": {
+      "portMappings": true
+    }
+  }]
+}
+EOF
+```
+
+### Container runtime
+
+This tutorial suggests downloading the containerd container runtime. When you
+set up containerd, it needs a lower-level container runtime such as `runc`.
+These steps also cover the setup steps you need to ensure that `runc` is
+also installed.
+
+**containerd**
+
+Visit [releases](https://github.com/containerd/containerd/releases) (external link).
+
+Download:
+
+{{< tabs name="download_containerd" >}}
+{{< tab name="x86-64" codelang="bash" >}}
+curl -LO https://github.com/containerd/containerd/releases/download/v1.7.22/containerd-1.7.22-linux-amd64.tar.gz
+{{< /tab >}}
+{{< tab name="ARM64" codelang="bash" >}}
+curl -LO https://github.com/containerd/containerd/releases/download/v1.7.22/containerd-1.7.22-linux-arm64.tar.gz
+{{< /tab >}}
+{{< /tabs >}}
+
+```shell
+curl -LO https://raw.githubusercontent.com/containerd/containerd/main/containerd.service
+```
+
+Configure:
+
+```shell
+sudo mkdir -p /etc/containerd
+
+sudo tee /etc/containerd/config.toml <<EOF
+version = 2
+
+[plugins."io.containerd.grpc.v1.cri".cni]
+
+    # binaries
+    bin_dir = "/opt/cni/bin"
+
+    # configure pod networking
+    conf_template = "/etc/cni/cni.template"
+EOF
+```
+
+Install:
+
+{{< tabs name="install_containerd" >}}
+{{< tab name="x86-64" codelang="bash" >}}
+sudo tar Cxzvf /usr/local/ containerd-1.7.22-linux-amd64.tar.gz
+{{< /tab >}}
+{{< tab name="ARM64" codelang="bash" >}}
+sudo tar Cxzvf /usr/local/ containerd-1.7.22-linux-arm64.tar.gz
+{{< /tab >}}
+{{< /tabs >}}
+
+```shell
+sudo cp containerd.service /etc/systemd/system/
+sudo systemctl daemon reload
```

Comment:
```suggestion
sudo systemctl daemon-reload
```

Reply:
Nice.
Line 327 needs to be updated also.

Reply:
... and line 474. Thanks!
Done